### PR TITLE
fix: Fix `rglplot(edge_label = )`

### LINKED ISSUE #25: rglplot edge labels bug
_From @gaborcsardi on July 2, 2014 17:24_

```
el <- cbind(sample(1:5), sample(1:5))
g <- graph.edgelist(el)
rglplot(g, edge.label = letters[1:ecount(g)])
```

gives 

```
Error in layout[, 4] : subscript out of bounds
```

_Copied from original issue: igraph/igraph#639_

### PR DESCRIPTION
**Pull request by maelle**
Fix #25

I do not understand the code, but this apparently fixes the bug and follows the pattern of the previous code lines. :innocent:

### CODE REVIEW

File: tests/testthat/test-plot.R [hunk 1]
```diff
@@ -65,3 +65,14 @@ test_that("basic plot test, spheres", {
     }
   )
 })
+
+test_that("rgplot() works", {
+  withr::local_envvar(RGL_USE_NULL = TRUE)
```
Comment (maelle) [inline tests/testthat/test-plot.R:70]: https://stackoverflow.com/a/46320771/5489251

Comment (krlmlr) [reply tests/testthat/test-plot.R:70]: Should this be a comment in the code instead? Only what's in the code will remain visible later.

Comment (krlmlr) [inline tests/testthat/test-plot.R:70]:
```suggestion
  # https://stackoverflow.com/a/46320771/5489251
  withr::local_envvar(RGL_USE_NULL = TRUE)
```

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**